doc/cs_registers.rst

```diff
@@ -117,6 +122,23 @@ Reset Value: ``0x0000_0000``
 | 4:0   | R   | **Exception Code**                                               |
 +-------+-----+------------------------------------------------------------------+
 
+When an exception is encountered, the corresponding exception code is stored in this register.
+
+
+Machine Trap Value (mtval)
+--------------------------
+
+CSR Address: ``0x343``
+
+Reset Value: ``0x0000_0000``
+
+When an exception is encountered, this register can hold exception-specific information to assist software in handling the trap.
+In the case of errors in the load-store unit ``mtval`` holds the address of the transaction causing the error.
+If this transaction is misaligned, ``mtval`` holds the address of the missing transaction part.
+In the case of illegal instruction exceptions, ``mtval`` holds the faulting instruction.
```

Comment:
How about making this an unordered list?

Reply:
Done.